client/src/pages/AddTrainee.js

```diff
@@ -5,20 +5,22 @@ import "../pages/addTrainee.css";
 export const AddTrainee = () => {
 	const [fullname, setFullName] = useState("");
 	const [githubUserName, setGithubUserName] = useState("");
+	const [cohortId, setCohortId] = useState("");
 	const navigate = useNavigate();
 
```

Comment:
I am believe we need to get cohort Id from url not from the form , we will fix tthis when we were sure about all the path

Reply:
Yes, in fact, we don't need to fill out cohort id as we know we are in the related cohort, but It's just because it's a primary key and  not null in the trainees' table, so when we add a trainee, we should fill it out to avoid error, maybe we could find a way to take it from URL and fill it as a default value